go.mod

```diff
@@ -135,6 +135,7 @@ require (
 	github.com/ettle/strcase v0.1.1 // indirect
 	github.com/fatih/structtag v1.2.0 // indirect
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
+	github.com/ghostiam/protogolint v0.1.0 // indirect
```

Comment:
your linter cannot be an indirect dependencies.

Reply:
This happened due to the call `go get -u`, fixed it using `go mod tidy`